server/handlers/requestHandlers.go

```diff
@@ -58,10 +68,73 @@ func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	expirationTime := time.Now().Add(time.Second * 20)
+
+	claims := &Claims{
+		Username: login.Email,
+		StandardClaims: jwt.StandardClaims{
+			ExpiresAt: expirationTime.Unix(),
+		},
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	tokenString, err := token.SignedString(jwtKey)
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	http.SetCookie(w,
+		&http.Cookie{
+			Name:    "token",
+			Value:   tokenString,
+			Expires: expirationTime,
+		})
+
 	log.Println("INFO [handlers/requestHandlers.go] Successful Log In")
 	http.Redirect(w, r, "/api/login_request_success", http.StatusSeeOther)
 }
 
 func (h *Handlers) LoginRequestSuccess(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "<h1>Successful Login!<h1>")
+	if h.TokenValid(w, r) {
+		fmt.Fprint(w, "<h1>Successful Login!<h1>")
+	}
+}
+
+func (h *Handlers) TokenValid(w http.ResponseWriter, r *http.Request) bool {
```

Comment:
In the structure that I'm familiar with, any function that returns something or doesn't directly or indirectly write to "w" should go in requestHelper.go as a private function (change TokenValid to tokenValid)